### PR TITLE
Add a foam sabre

### PR DESCRIPTION
**Pull request by Radezolid**
## About the PR
Adds a foam sabre, it can spawn in maints and mimes can select to spawn with it.
Also adds a sharp foam sabre, only way to obtain is through the syndicate uplink.

## Why / Balance
Fun things for people to play + a stealth weapon, it's damage is equal to the cane blade, same attack rate too.

## Technical details
Added YAMLs, sprites and locales.

## Media

image
image
image
image

Evil version:
image
image
image

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

None.

**Changelog**

:cl:
- add: Added foam sabres! They may appear in maintenance lockers and mimes can select to spawn with it in the loadout selection.
- add: Added a sharp version of the foam sabre toy, purchasable from the syndicate uplink at 6 TC with a possible discount to 4 TC.

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Melee/foam_blade.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+- type: entity
+  parent: FoamSabre
+  id: RealFoamSabre
+  name: foam sabre
+  suffix: Real
+  description: Cosplay as a captain and command your group of friends!. It looks sturdier than usual.
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Melee/foam_blade.yml]: !. -> !

File: Resources/Textures/DeltaV/Clothing/Belt/foamsheath.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "directions": 4
+    },
+    {
+        "name": "sheath-sabre"
+    },
+    {
+        "name": "sheath"
+    }
+  ]
+}
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Clothing/Belt/foamsheath.rsi/meta.json]: keep indent consistent at 4 spaces (the amazing fun time of not just using tabs like a sane project)

**Comment by CybersunOperative** [comment]
Hi. Hello. Val here. The other curators that weighed in on this said they liked it. I do have some minor nitpicks though.
I would highly recommend removing the "It looks sturdier than most." from the evil foam saber description, as that immediately gives it away as le evil valid toy. 
You also have an extra period after "A real foam sheathe to cosplay as the captain!" 
Otherwise, it seems good. I like the 1% chance for the toy to reflect. Generally a very cute addition to the foam weapon set.

**Comment by Radezolid** [comment]
> I would highly recommend removing the "It looks sturdier than most." from the evil foam saber description, as that immediately gives it away as le evil valid toy.

That was kind of my intention, a foam weapon being able to do 14 slash kinda feels like it needs to be able to be noticed somehow, it isn't like a stealth pen that it actually hides how it works, the evil version is just a blade with the foam on top as disguise, but sure, I don't really mind changing that.

Regarding the extra period, it's a thing in my native language and sometimes i forgor when writing

**Comment by deltanedas** [comment]
description doesnt matter since it has a damage examine compared to foam sabre which doesnt

**Comment by Radezolid** [comment]
Yup, it was flavor text to give a reason for the people to do the "this ones does damage, evil."

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Textures/DeltaV/Clothing/Belt/foamsheath.rsi/meta.json (hunk 2 above)
Comment (Radezolid) [reply Resources/Textures/DeltaV/Clothing/Belt/foamsheath.rsi/meta.json]: Last time I copy paste a meta.json
Will fix once I'm able.

**Review by deltanedas — DISMISSED** [review]

**Comment by Monotheonist** [comment]
Make sure to add it to the sword mail in `mail.yml.` Otherwise no one's gonna get the foam sabre in the mail

**Comment by Radezolid** [comment]
Hadn't thought about giving it on the mail, thanks for the idea

**Review by deltanedas — APPROVED** [review]

**Pull request merged**